setup.py

```diff
@@ -18,9 +16,13 @@
 else:
     extra_compile_args = []
 
+exec_results = {}
+exec(open(path.join(path.dirname(__file__), 'fuel/version.py')), exec_results)
```

Comment:
Nitpick, but I prefer using `with` instead of relying on the GC to close this file handle.